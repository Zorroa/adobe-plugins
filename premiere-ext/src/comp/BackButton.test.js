import renderer from "react-test-renderer"
import BackButton from "./BackButton"

const history = { goBack: function () {} }

describe("<BackButton/>", () => {
  it("should render properly", () => {
    const wrapper = renderer.create(<BackButton history={history} />)
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
