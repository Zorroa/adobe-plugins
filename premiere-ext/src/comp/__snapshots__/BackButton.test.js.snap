// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackButton/> should render properly 1`] = `
<button
  className="btn btn-dark btn-md btn btn-primary"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  <svg
    className="bi bi-arrow-left-circle-fill"
    fill="currentColor"
    height="1em"
    viewBox="0 0 18 18"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-4.5.5a.5.5 0 0 0 0-1H5.707l2.147-2.146a.5.5 0 1 0-.708-.708l-3 3a.5.5 0 0 0 0 .708l3 3a.5.5 0 0 0 .708-.708L5.707 8.5H11.5z"
      fillRule="evenodd"
    />
  </svg>
  Â Go Back
</button>
`;
